.github/workflows/pull_request_test.yml

```diff
@@ -0,0 +1,15 @@
+name: Automated testing workflow
+on: pull_request
```

Comment:
I think it would make sense to also run on pushes to any branch (that would catch errors of codeowners before they make the PR).

Reply:
Will add that

Reply:
Since I have not mentioned any branch filter for the action, this will automatically include all branches as per <a href="https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet">Filter pattern cheat sheet</a>

Reply:
I think you're misunderstanding.  I'm not talking about PRs to any branch, I'm talking about pushes to branches From that page:
```
on: [push, pull_request]
```

Reply:
Oh! my bad will change